source/linux/Foundational_Components/Kernel/Kernel_Drivers/QSPI.rst

```diff
@@ -420,11 +420,248 @@ or via command line arguments):
    $ dd if=/dev/mtd6 of=tmp_read.txt bs=num count=1 # read to num bytes to flash
    $ diff tmp_read.txt tmp_write.txt # should be NULL
 
+.. note::
+
+   Make sure UBIFS filesystem is enabled in the kernel (refer to `this
+   section <#enabling-qspi-driver-configurations>`__ for more information).
+
+.. linux_ubifs_start
+
+.. rubric:: Unsorted Block Image File System (UBIFS)
+   :name: linux-ubifs
```

Comment:
Having a named rubric breaks when using include directives. Either remove the include referencing this file or the name from this rubric.

Reply:
Don't like includes either. 
Create a separate file called ubifs.rst and link it to all the flash specific guides as hyperlinks